language/move-lang/src/errors/diagnostic_codes.rs

```diff
@@ -167,6 +175,8 @@ impl DiagnosticInfo {
 }
 
 impl Severity {
+    pub const MAX: Self = Self::Warning;
```

Comment:
Maybe it's just me, but I found it odd that the `MAX` member of the `Severity` type is `Warning`. It made me do a double-take because wouldn't the "maximum" severity be a blocking error?

Looking below at how this member is used for the "default" severity, I wonder if it ought to be renamed to something besides `MAX`? Or, more likely, I'm misunderstanding something about this code... 😅

Reply:
I went with `MAX` in the orderable sense, that is for any `x: Severity` `x < Severity::MAX`. I tried to match the naming for `u64` and the like, but it definitely is a bit confusing with the word `Severity` as those tend to have their own meaning. 

Maybe I should flip all of these orderings? Might make more sense to have the *least severe* diagnostic type be ordered the least. Any thoughts?

Reply:
I'm actually so convinced that this is the right switch that I'm going to do that. Hopefully I don't regret not waiting for a response, lol

Reply:
Nice! I think that makes sense. (Sorry, I'm still trying to convince GitHub to send me push notifications...)

Reply:
No need to be sorry, lol. I sent that message super late your time and responded to it soon after. But glad it makes sense now ;) ... hopefully